tests/qos/tunnel_qos_remap_base.py

```diff
@@ -394,13 +370,25 @@ def update_docker_services(rand_selected_dut, swap_syncd, disable_container_auto
     for service in SERVICES:
         _update_docker_service(rand_selected_dut, action="stop", **service)
 
+    asic = rand_selected_dut.get_asic_name()
```

Comment:
Hi @bingwang-ms , the update_docker_services fixture is only referenced by fixture setup_module, which is invoked only by test cases test_xoff_for_pcbb and test_tunnel_decap_dscp_to_pg_mapping. But I think the disable aging is required by all the pfc related cases.
Have you run all the tests on Nvidia platforms with this change, did they all pass?

Reply:
That's a good catch. I only manually ran `test_xoff_for_pcbb` to verify the change. 
So probably we need to keep the autoused fixture to ensure the script is copied no matter whether `swap_syncd` is enabled or not?

Reply:
I suggest we keep the disable_packet_aging autouse fixture and only change the scope to "class", then it should work with either syncd swapped or not.